README.md

```diff
@@ -16,3 +16,11 @@ You can also configure the port (with `-p`) and root directory (with
 ```bash
 java duckling.jar -p $(YOUR_FAVORITE_PORT) -d $(YOUR_FAVORITE_DIR)
 ```
+
+## A small overview
+
+Right now Duckling doesn't support routing at all.  It will display some HTML as expected, provide a small navigable directory listing, and respond with a 404 to missing content.
+
+## Known bugs
+
+* Deep traversal of directories is currently not supported.  This is getting punted on as of release 0.0.1 in the interests of getting a more thoroughly tested `FolderContents` and usable responder in play, vs. upgrading the current one further (which is moot, because it's serving its purpose as-is).
```

Comment:
Let's track known issues in Github issues instead of the README - ideally we have 0 known bugs at all times. Of course some will slip through, but putting it into the code itself seems kind of high-maintenance since that hopefully changes very quickly. The line between bugs & features can also get pretty important on some projects where the client doesn't pay for bug fixes (and so we need to be pretty clear & communicative about requirements for stories before committing to estimates).